lib/rest-adapter.js

```diff
@@ -258,6 +259,10 @@ RestAdapter.prototype.createHandler = function() {
 
   var handleUnknownPaths = this._shouldHandleUnknownPaths();
 
+  // Mount rest-router
+  var customRouter = new RestRouter();
+  root.use(customRouter.handle());
```

Comment:
I find this weird, why can't we directly `root.use(customRouter)`? I'd like our new router be as much compatible with the built-in Express router as possible.